### ISSUE TITLE
Smart Assist does not appear in Conversation Summary form

### ISSUE THREAD

**Issue by MTH2019** [microsoft/Dynamics365-Apps-Samples#4]
I've followed the instructions and published the bot code back to Azure. I've followed the Smart Assist Agent instructions from Microsoft--adding the Application Bot User with all the links. I've added the Smart Assist Control to the Conversation Summary form and published. Yet the Smart Assist Bot does not appear.
Any suggestions on troubleshooting this.
Thanks
![SmartAssist on Conversation Form in CE](https://user-images.githubusercontent.com/49790193/69350406-7c8c4400-0c47-11ea-92c9-72e1454de9a2.jpg)

**Comment by sudhkaoc** [comment]
Does your bot work in isolation?(When tested in web chat for example). The control shows the adaptive cards the bot respond with. Do make sure the bot correctly responds as expected by testing it in bot emulator or in web chat.
Also make sure you have enabled "Teams" channel for the bot.
If there are any "issues" that show up in the Teams channel, post back here as what it is.

**Comment by MTH2019** [comment]
I just started from scratch. Went through all of the changes to the solution and published back to Azure. The "Test in Web Chat" does work and shows the initial card of "Welcome to Bot Framework!" If I type in help, it responds.
I've added the Teams Channel. However, I wasn't sure if I need to publish in my Org's "Teams Tenant App Catalog" as it states under the Publish tab.
I've also create the App Registration as shown in the link "Build web applications using Server-to-Server(S2S) authentication"
Created a user and added in the Omnichannel admin area with role of Omnichannel Agent.
Added Smart Assist control to Conversation Title field and added to form.
Still the Agent does not show in Omnichannel.

Can't imagined what I missed. I followed all of your steps (including updating the appsettings and startup.cs. Solution builds fine and I published.

Thanks
